extensions/stackdriver/common/constants.h

```diff
@@ -17,6 +17,52 @@ namespace Extensions {
 namespace Stackdriver {
 namespace Common {
 
+// Measure names of metrics.
+constexpr char kServerRequestCountMeasure[] =
```

Comment:
`constexpr absl::string_view` is easier to use for constant strings.

Reply:
Why it is easier to use? Just replace the type seems to be not working, for example the following statement won't work, protobuf map cannot use absl::string_view as a key:
```
(*monitored_resource.mutable_labels())[kProjectIDLabel]
```

Reply:
OK, fair enough. Leave as-is, until protobuf adopts string_view. It's easier because it gives you all the string functions (concat, size, etc).